src/wallet/asyncrpcoperation_mergetoaddress.cpp

```diff
@@ -825,10 +825,10 @@ UniValue AsyncRPCOperation_mergetoaddress::perform_joinsplit(
     UniValue arrInputMap(UniValue::VARR);
     UniValue arrOutputMap(UniValue::VARR);
     for (size_t i = 0; i < ZC_NUM_JS_INPUTS; i++) {
-        arrInputMap.push_back(inputMap[i]);
+        arrInputMap.push_back(static_cast<uint64_t>(inputMap[i]));
```

Comment:
I would mildly prefer the "extend UniValue to support ``size_t``" approach, but I don't object to these casts since we can clean it up later.